src/lib/execution.ts

```diff
@@ -69,6 +69,10 @@ export function calculateManualExecProof(
       )
     }
     // Hash leaf node
+    // TODO: msg here could be of any type (1.5 or 1.6), but we are using a specific version hasher. This will throw at runtime if encounters different message versions
```

Comment:
@andrevmatos I saw this issue when adding the Aptos type. As I mention in the comment, I'm not sure if we want to accept any type of CCIPMessage in the batch. If so, we should have the specific version hasher for each message version, if not, we should type `messagesInBatch` to accept only one version.

I test the issue [here](https://github.com/smartcontractkit/ccip-tools-ts/pull/19/files#diff-52fd748922ee0f1d394f7725994a3ed32265ba8f20405e25ab0e6230060c4366R166)